library/src/main/java/com/bumptech/glide/load/resource/bitmap/RecyclableBufferedInputStream.java

```diff
@@ -368,15 +368,15 @@ public synchronized void reset() throws IOException {
    */
   @Override
   public synchronized long skip(long byteCount) throws IOException {
```

Comment:
swapped order to mimic java.io.InputStream#skip (i.e. don't throw when no bytes are skipped)